stdsimd/arch/detect/os/aarch64.rs

```diff
@@ -0,0 +1,64 @@
+//! Run-time feature detection for Aarch64 on any OS that traps the mrs instruction.
```

Comment:
Perhaps this file could go coresimd as it would be useful for bare metal too?

Reply:
@parched we don't really have a mechanism to expose this from `core::arch` yet (and it is unclear how to do that). I'd leave doing that to "future work".